docs/guide.md

```diff
@@ -73,6 +73,17 @@ See also [Setting the permission policy](http://www.django-rest-framework.org/ap
     permission_classes = [HasAPIKey | IsAuthenticated]
     ```
 
+### Manually validating API keys
```

Comment:
I think we should move this to the bottom of the page, as a last section. It's a "manual mode when you need to validate yourself" so it makes sense to keep it off the "happy I-only-use-DRF-permissions" path, WDYT?

Reply:
I don’t mind, I just put it where I thought I’d go looking for it. :)